data/week1/4-reading-input.md

```diff
@@ -52,11 +52,12 @@ public class Program {
 }
 ```
 
-More precisely, input is read with the `scanner` tool's `nextLine()` reading method. The call `scanner.nextLine()` is left waiting for the user to write something. When user writes something and presses enter, the provided string is assigned to a the string variable `message`. The program is then able to reference the variable `message` later on, for instance to print the message. If you are running a program like this, you should enter a message in the console as if you were the user. The `scanner.nextLine();` command reads the user's input and *returns* a string. If we then want to use the string in the program, it must be saved to a string variable -- `String message = scanner.nextLine();`. A value saved to a variable can be used repeatedly.
+
+Input is read using the `scanner` tool's `nextLine()` method. The `nextLine()` call waits for the user to enter text and press enter. The entered text is then assigned to the string variable `message`. The program can reference the `message` variable later, for example to print it. When running the program, enter a message in the console as if you were the user. The `nextLine()` command reads the input and *returns* it as a string. To use the returned string in the program, it must be saved to a string variable (e.g. `String message = scanner.nextLine();`). A value saved in a variable can be reused.
```

Comment:
The entered text is then assigned to a `String` type variable named `message`, which is the variable type used in Java to store text.

Reply:
Later ook de 'string' vervangen met `String`